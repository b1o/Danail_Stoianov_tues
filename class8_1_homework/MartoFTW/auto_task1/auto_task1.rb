require "csv"

m_name = ARGV[1]
arr_out = Array.new

	CSV.foreach(ARGV[0]) do |row|
		if row[1] == m_name
			arr_out << row
		end
	end

	arr_out.sort!{|a,b| a[2]<=>b[2]}

	CSV.open("auto_task1_result.csv","w") do |csv|
		arr_out.each do |elem|
			csv << elem
		end
	end

