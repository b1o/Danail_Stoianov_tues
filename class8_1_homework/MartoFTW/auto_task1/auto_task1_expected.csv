14/10/2013,Renault,16.70,Aleksandyr Petrov
15/10/2013,Renault,23.43,Petar Grigorov
10/10/2013,Renault,24.00,Aleksandyr Petrov
14/10/2013,Renault,59.99,Marian Arianov
